x/aol/client/cli/txTopic.go

```diff
@@ -1,93 +1,36 @@
 package cli
 
 import (
-	"github.com/spf13/cobra"
-	"strconv"
-
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
 	"github.com/medibloc/panacea-core/x/aol/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagDescription = "description"
+	flagMoniker     = "moniker"
+	flagFeePayer    = "payer"
 )
 
 func CmdCreateTopic() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create-topic [description] [totalRecords] [totalWriter]",
+		Use:   "create-topic [topicName]",
 		Short: "Creates a new topic",
-		Args:  cobra.ExactArgs(3),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			argsDescription := string(args[0])
-			argsTotalRecords, _ := strconv.ParseInt(args[1], 10, 64)
-			argsTotalWriter, _ := strconv.ParseInt(args[2], 10, 64)
-
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			msg := types.NewMsgCreateTopic(clientCtx.GetFromAddress().String(), string(argsDescription), int32(argsTotalRecords), int32(argsTotalWriter))
-			if err := msg.ValidateBasic(); err != nil {
-				return err
-			}
-			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
-		},
-	}
-
-	flags.AddTxFlagsToCmd(cmd)
-
-	return cmd
-}
-
-func CmdUpdateTopic() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "update-topic [id] [description] [totalRecords] [totalWriter]",
-		Short: "Update a topic",
-		Args:  cobra.ExactArgs(4),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			id, err := strconv.ParseUint(args[0], 10, 64)
-			if err != nil {
-				return err
-			}
-
-			argsDescription := string(args[1])
-			argsTotalRecords, _ := strconv.ParseInt(args[2], 10, 64)
-			argsTotalWriter, _ := strconv.ParseInt(args[3], 10, 64)
-
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			msg := types.NewMsgUpdateTopic(clientCtx.GetFromAddress().String(), id, string(argsDescription), int32(argsTotalRecords), int32(argsTotalWriter))
-			if err := msg.ValidateBasic(); err != nil {
-				return err
-			}
-			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
-		},
-	}
-
-	flags.AddTxFlagsToCmd(cmd)
-
-	return cmd
-}
-
-func CmdDeleteTopic() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "delete-topic [id] [description] [totalRecords] [totalWriter]",
-		Short: "Delete a topic by id",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			id, err := strconv.ParseUint(args[0], 10, 64)
+			clientCtx, err := client.GetClientTxContext(cmd)
 			if err != nil {
 				return err
 			}
 
-			clientCtx, err := client.GetClientTxContext(cmd)
+			description, err := cmd.Flags().GetString(flagDescription)
```

Comment:
`description` is optional?

Reply:
yeah. it has been optional: https://github.com/medibloc/panacea-core/blob/985271f795c2ff0bf1f6b63152d97fabb41dbfa2/x/aol/client/cli/tx.go#L79

Reply:
I'll change codes to use `viper.GetString()`.